code/modules/mob/mob.dm

```diff
@@ -714,7 +714,7 @@ var/next_mob_id = 0
 			layer = initial(layer)
 	update_transform()
 	update_action_buttons_icon()
-	if(istype(src, /mob/living))
```

Comment:
istype(src) ....istype(src) ...;NGhdfijiugdifjgisgokjgk

but it's not your fault so okay.